scripts/cleanup_clusters/cleanup_clusters.sh

```diff
@@ -30,6 +30,10 @@ fi
 # Leonardo only creates clusters in this region
 CLUSTER_REGION='us-central1'
 
+# NOTE! this assumes the BSD date command on OS X.  Will likely need tweaks for GNU date on Linux.
+TODAY=$(date +"%Y-%m-%d")
```

Comment:
```
~ $ date +"%Y-%m-%d"
2018-02-23
```

Reply:
So Linux date works the same?  I'll remove the comment there then.